src/aircloak/JsonApi/ResponseTypes/Columns/AircloakColumn.cs

```diff
@@ -0,0 +1,19 @@
+namespace Aircloak.JsonApi.ResponseTypes
+{
+    /// <summary>
+    /// Abstract base class for Diffix column values which can be suppressed or NULL.
+    /// </summary>
+    /// <typeparam name="T">The expected type of the column value.</typeparam>
+    public abstract class AircloakColumn<T>
```

Comment:
The generic parameter is not used inside the class. Could it be removed and added just in the required derived class?

Reply:
I wasn't sure about this - it's true the parameter isn't used in the abstract class however if we keep it, it ensures that the columns are always tagged with the expected return type. If a column value is suppressed or null, that doesn't mean we lose the type information - so it should be useful to keep this and allow the type-checker to ensure that we don't mix columns of different types. What do you think?